airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -28,6 +28,7 @@ acceptance_tests:
       timeout_seconds: 2400
       expect_records:
         path: integration_tests/expected_records.jsonl
+      fail_on_extra_columns: false
```

Comment:
lol, this was the only 'clean' change

Reply:
I've learned that this is the key to reviewing changes like these! 
<img width="243" alt="image" src="https://user-images.githubusercontent.com/22333512/227818422-d95bead9-09cd-4251-8e5d-3781efdfb67f.png">